src/ManageCourses.Domain/Migrations/ManageCoursesDbContextModelSnapshot.cs

```diff
@@ -242,15 +242,15 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<DateTime>("CreatedUtc")
                         .HasColumnName("created_utc");
 
-                    b.Property<string>("Email")
-                        .HasColumnName("email");
+                    b.Property<int?>("McUserId")
```

Comment:
I think you can get this to not be nullable by adding `[Required]` to the McUser property on the session model